beacon-chain/state/stategen/service.go

```diff
@@ -187,3 +191,29 @@ func (s *State) finalizedState() state.BeaconState {
 	defer s.finalizedInfo.lock.RUnlock()
 	return s.finalizedInfo.state.Copy()
 }
+
+func initFieldMap() {
```

Comment:
The field map initialization inside `beacon-chain/state/state-native/types.go` is needed for tests. However, at the time when the code inside `init()` runs we don't have features configured yet. The initialization here is ran after features are configured.